pkgs/tools/misc/xsel/default.nix

```diff
@@ -1,13 +1,22 @@
-{stdenv, fetchurl, xlibsWrapper}:
+{stdenv, fetchFromGitHub, xlibsWrapper, autoreconfHook }:
 
-stdenv.mkDerivation {
-  name = "xsel-1.2.0";
-  src = fetchurl {
-    url = http://www.vergenet.net/~conrad/software/xsel/download/xsel-1.2.0.tar.gz;
-    sha256 = "070lbcpw77j143jrbkh0y1v10ppn1jwmjf92800w7x42vh4cw9xr";
+stdenv.mkDerivation rec {
+  name = "xsel-${version}";
+
+  version = "git-2016-09-02";
+
+  src = fetchFromGitHub {
+    owner = "kfish";
+    repo = "xsel";
+    rev = "aa7f57eed805adb09e9c59c8ea841870e8206b81";
+    sha256 = "04mrc8j0rr7iy1k6brfxnx26pmxm800gh4nqrxn6j2lz6vd5y9m5";
   };
 
-  buildInputs = [xlibsWrapper];
+  buildInputs = [ xlibsWrapper autoreconfHook ];
```

Comment:
If you also got rid of `xlibsWrapper`, it would be nice. I had only left it defined because so many packages were using it.